test_run_handler.go

```diff
@@ -16,71 +16,26 @@ package wptdashboard
 
 import (
 	"encoding/json"
-	"fmt"
-	"io/ioutil"
 	"net/http"
 	"sort"
-	"time"
 
 	"google.golang.org/appengine"
 	"google.golang.org/appengine/datastore"
 )
 
-func testRunHandler(w http.ResponseWriter, r *http.Request) {
+// testRunsHandler handles GET/POST requests to /test-runs
+func testRunsHandler(w http.ResponseWriter, r *http.Request) {
 	if r.Method == "POST" {
-		handlePost(w, r)
+		// /test-runs is the legacy POST endpoint, migrated to /api/runs and left to avoid breakages.
```

Comment:
Can you make a bug and add `TODO(#n)` where `n` is the bug #?

Reply:
Done.